packages/desktop-electron/index.ts

```diff
@@ -399,3 +400,18 @@ ipcMain.on('set-theme', (_event, theme: string) => {
     );
   }
 });
+
+ipcMain.handle(
+  'move-budget-directory',
+  async (_event, currentBudgetDirectory: string, newDirectory: string) => {
+    try {
+      await copy(currentBudgetDirectory, newDirectory, {
+        overwrite: true,
+      });
+      await remove(currentBudgetDirectory);
+    } catch (error) {
+      console.error('There was an error moving your directory', error);
+      return Promise.reject(error);
+    }
+  },
+);
```

Comment:
Copies the budget files into the new directory, then removes the old directory. If it fails the copy nothing is deleted. If it fails the delete the copy has been successful and it's recoverable - user is instructed to check the directories.